client/js/components/ParkFeatureListItem.jsx

```diff
@@ -4,18 +4,23 @@ import icons from '../utils/icons.json';
 
 
 export default class ParkFeatureListItem extends React.Component {
+    onClick() {
+        this.props.selectFeature(this.props.id);
+        this.props.slideUp();
```

Comment:
i noticed this slide/up/down thing when doing the routing. currently there wasnt a simple way i could see if having the router router to the up state. something to think about